.gitignore

```diff
@@ -28,6 +28,7 @@ __pycache__
 .tox
 nosetests.xml
 unit_tests/testdata.json
+*output_sanitize*
```

Comment:
Why not the exact path `docs/source/output_sanitize.cfg`?  Where else could that file be anyways?  Let's not ignore too much and miss a real important file.